### PR TITLE
Compute filter scatter [VS-392]

### PR DESCRIPTION
**Pull request by rsasch**
Closes https://broadworkbench.atlassian.net/browse/VS-392

### CODE REVIEW

**Review by mcovarr — APPROVED** [review]

File: scripts/variantstore/wdl/GvsQuickstartIntegration.wdl [hunk 1]
```diff
@@ -64,7 +64,6 @@ workflow GvsQuickstartIntegration {
             input_vcfs = input_vcfs,
             input_vcf_indexes = input_vcf_indexes,
             filter_set_name = "quickit",
-            create_filter_set_scatter_count = 20,
```
Comment (RoriCremer) [inline scripts/variantstore/wdl/GvsQuickstartIntegration.wdl:67]: does this mean we want to remove it from the quickstart instructions?

**Review by RoriCremer — APPROVED** [review]

File: scripts/variantstore/wdl/GvsCreateFilterSet.wdl [hunk 2]
```diff
@@ -70,6 +69,11 @@ workflow GvsCreateFilterSet {
       project_id = project_id
   }
 
+  Int scatter_count = if GetNumSamplesLoaded.num_samples < 100 then 20
```
Comment (gbggrant) [inline scripts/variantstore/wdl/GvsCreateFilterSet.wdl:72]:
```suggestion
  Int scatter_count = if GetNumSamplesLoaded.num_samples < 100 then 20
                      else if GetNumSamplesLoaded.num_samples < 1000 then 100
                      else if GetNumSamplesLoaded.num_samples < 10000 then 200
                      else if GetNumSamplesLoaded.num_samples < 100000 then 500
                      else 1000```

Comment (gbggrant) [reply scripts/variantstore/wdl/GvsCreateFilterSet.wdl:72]: Might be a little cleaner?

**Review by gbggrant — APPROVED** [review]

File: scripts/variantstore/wdl/GvsQuickstartIntegration.wdl (hunk 1 above)
Comment (rsasch) [reply scripts/variantstore/wdl/GvsQuickstartIntegration.wdl:67]: good idea — done

File: scripts/variantstore/wdl/GvsCreateFilterSet.wdl (hunk 2 above)
Comment (rsasch) [reply scripts/variantstore/wdl/GvsCreateFilterSet.wdl:72]: I went with the formatting that the Winstanley plugin recommended, although I like the way your suggestion looks,.

**Pull request merged**